R/add_prop_ci.R

```diff
@@ -0,0 +1,168 @@
+#' Add Proportion CIs
+#'
+#' Add a new column with the confidence intervals for proportions.
+#'
+#' @param x A `tbl_summary` object
+#' @param pattern String indicating how the confidence interval will be placed.
+#' Default is `"{conf.low}%, {conf.high}%"`
+#' @param method Confidence interval method. Must be one of
+#' `c("wilson", "wilson.no.correct", "exact", "asymptotic")`. Default is `"wilson"`.
+#' Methods `c("wilson", "wilson.no.correct")` are calculated with `prop.test()`,
```

Comment:
Maybe it could be relevant to add a Details section explaining the different methods, ie. that no correct is without Yate's continuity correction, that asymptotic is also known as Wald test, and here without Yate's CC and a quick explanation of asymptotic. Maybe add one or two references.